src/actions/Action.ts

```diff
@@ -7,6 +7,7 @@ class Action implements IAction {
   // addParam(w_100).addParam(w_200) should result in w_200. and not w_100,w_200
   params: Map<string, Param> = new Map();
   delimiter = ','; // {param}{delimiter}{param} for example: `${'w_100'}${','}${'c_fill'}`
+  prepareParam() {}
```

Comment:
I don't think prepareParam should be in Action, but we can refactor this later.
@patrick-tolosa what do you think

Reply:
This is the equivalent of `build` in the other SDKs.

The problem we have here is that we can create the final string only when we have all the information, regardless of the order of insertion.

Notice how `prepareParam` contains action-specific business logic, and it's not something generic. 

We could have it as an external function `prepareBorderParam` but I don't see much merit in spreading the logic to more files